packages/block-editor/src/hooks/block-bindings-panel.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { store as blocksStore } from '@wordpress/blocks';
+import {
+	PanelBody,
+	__experimentalItemGroup as ItemGroup,
+	__experimentalItem as Item,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../store';
+import { unlock } from '../lock-unlock';
+import InspectorControls from '../components/inspector-controls';
+
+export const BlockBindingsPanel = () => {
+	const { block } = useSelect( ( select ) => {
+		const { getSelectedBlockClientId, getBlock } = unlock(
+			select( blockEditorStore )
+		);
+
+		const _selectedBlockClientId = getSelectedBlockClientId();
+
+		return {
+			selectedBlockClientId: _selectedBlockClientId,
+			block: getBlock( _selectedBlockClientId ),
+		};
+	}, [] );
+
+	const bindings = block?.attributes?.metadata?.bindings;
+
+	const sources = useSelect( ( select ) =>
+		unlock( select( blocksStore ) ).getAllBlockBindingsSources()
+	);
+
+	const hasBindings = block?.attributes?.metadata?.bindings
+		? Object.keys( block.attributes.metadata.bindings ).length > 0
+		: false;
```

Comment:
Can't we use the `bindings` property here? If we do that maybe we don't need this `hasBindings` at all.

Reply:
Good catch — this has been updated